vars/sconsBuild.groovy

```diff
@@ -189,12 +188,12 @@ def call(Map config = [:]) {
     if (config['skip_clean']) {
         clean_cmd += "echo 'skipping scons -c'\n"
     } else {
-        def clean_files = "_build.external"
+        def clean_files = "install"
```

Comment:
This wouldn't be backward compatible with building an older commit of daos would it?

Reply:
We dropped _build.external about a year ago.

Reply:
> We dropped _build.external about a year ago.

It doesn't matter how long ago it was.  If somebody checked out a PR from over a year ago, the clean would fail, where it would have worked when it was originally built.

Reply:
Do we support builds of DAOS from that long ago?  I can easily back out this part but I'd be amazed if that worked.

Reply:
Just to answer the question...

> Do we support builds of DAOS from that long ago?

I haven't been told otherwise.  Sounds like a policy decision issue to me.  But note that this would break being able to build even our current (1.0.x) release.

> I can easily back out this part but I'd be amazed if that worked.

Maybe, maybe not, because we certainly don't actively test changes for API breakage.  Probably we could/should.  As it is, we just try to do our best with our eyeballs.  It's entirely possible that if somebody tried to build our 1.0.x release in CI it would break and we'd have to patch up the pipeline-lib API.  But we should avoid breaking it when our eyeballs show us it is getting broken.

I'm in fact un-breaking pipeline-lib API at this very moment, that landed yesterday or the day before.